test/controllers/games_controller_test.rb

```diff
@@ -0,0 +1,13 @@
+require "test_helper"
+
+class GamesControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get games_index_url
+    assert_response :success
+  end
+
+  test "should get -–skip-routes" do
```

Comment:
I think we can omit this test as we need to be specific on what route action we want to test.

Reply:
Removed in https://github.com/amykingOW/hangman/pull/1/commits/932c149f90efd4cb48318eb772123227b477f2d2.